pkg/porter/plugins.go

```diff
@@ -177,3 +190,44 @@ func (p *Porter) UninstallPlugin(ctx context.Context, opts pkgmgmt.UninstallOpti
 
 	return nil
 }
+
+func (p *Porter) getInstallConfigs(ctx context.Context, opts plugins.InstallOptions) ([]pkgmgmt.InstallOptions, error) {
```

Comment:
Nit: Call this maybe getPluginsInstallConfigs so we don't confuse this with "Installations" down the road